src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -99,14 +102,36 @@ private void parse(String [] argv) {
 		if ( options.has("log_level")) {
 			this.log_level = parseLogLevel((String) options.valueOf("log_level"));
 		}
+
 		if ( options.has("host"))
-			this.mysqlHost = (String) options.valueOf("host");
+			this.maxwellMysql.mysqlHost = (String) options.valueOf("host");
 		if ( options.has("password"))
-			this.mysqlPassword = (String) options.valueOf("password");
+			this.maxwellMysql.mysqlPassword = (String) options.valueOf("password");
 		if ( options.has("user"))
-			this.mysqlUser = (String) options.valueOf("user");
+			this.maxwellMysql.mysqlUser = (String) options.valueOf("user");
 		if ( options.has("port"))
-			this.mysqlPort = Integer.valueOf((String) options.valueOf("port"));
+			this.maxwellMysql.mysqlPort = Integer.valueOf((String) options.valueOf("port"));
+
+		if ( options.has("replication_host"))
+			this.replicationMysql.mysqlHost = (String) options.valueOf("replication_host");
```

Comment:
this could be DRY'ed, something like 
``
  class MaxwellMysqlConfig {  
    public void parseOptions(String prefix, Options options) { 
      if ( options.has(prefix + "host") ) {
         ...
      }
   }
 }
```